src/data/data.JSON

```
@@ -14619,5 +14619,166 @@
       ],
       "label": "distribution"
     }
+  },
+  {
+    "company": {
+      "data": "Red Hat",
+      "label": "company"
+    },
+    "system": {
+      "data": "Patternfly Design System",
+      "label": "system",
+      "url": "https://www.patternfly.org/",
+      "$addedAt": "202004221115"
+    },
+    "repository": {
+      "data": "GitHub",
+      "label": "repository",
+      "url": "https://github.com/patternfly/patternfly"
+    },
+    "codeDepth": {
+      "data": "HTML/CSS/JS/TS",
+      "label": "code depth"
+    },
+    "components": {
+      "data": "yes",
+      "label": "components"
+    },
+    "js": {
+      "data": "Patternfly",
+      "label": "JS library/framework",
+      "url": "https://www.patternfly.org/v4/get-started/developers#htmlcss"
+    },
+    "ts": {
+      "data": "yes",
+      "label": "typescript"
+    },
+    "webComponents": {
+      "data": "yes",
+      "label": "web components"
+    },
+    "tests": {
+      "data": [
+        "BackstopJS",
+        "jest",
+        "Web Component Tester"
+      ],
+      "label": "tests"
+    },
+    "linter": {
+      "data": "TSLint",
+      "label": "linter"
+    },
+    "css": {
+      "data": [
+        "Sass",
+        "PostCSS"
+      ],
+      "label": "CSS"
+    },
+    "cssInjs": {
+      "data": "no",
+      "label": "CSS in JS"
+    },
+    "designTokens": {
+      "data": "no",
+      "label": "design tokens"
+    },
+    "bundleManager": {
+      "data": [
+        "Webpack",
+        "Gulp"
+      ],
+      "label": "bundle manager"
+    },
+    "uiKit": {
+      "data": "yes",
+      "label": "UI kit",
+      "url": "https://www.patternfly.org/v4/design-guidelines/styles/colors"
+    },
+    "brandGuidelines": {
+      "data": "yes",
+      "label": "brand guidelines",
+      "url": "https://www.patternfly.org/v4/design-guidelines/styles/colors"
+    },
+    "colorPalette": {
+      "data": "yes",
+      "label": "color palette",
+      "url": "https://www.patternfly.org/v4/design-guidelines/styles/colors#color-palette"
+    },
+    "colorNaming": {
+      "data": "no data",
+      "label": "color naming"
+    },
+    "contrastAnalysis": {
+      "data": "yes",
+      "label": "contrast analysis",
+      "url": "https://www.patternfly.org/v4/design-guidelines/styles/colors#contrast-ratios"
+    },
+    "typography": {
+      "data": "yes",
+      "label": "typography",
+      "url": "https://www.patternfly.org/v4/design-guidelines/styles/typography"
+    },
+    "icons": {
+      "data": "yes",
```

```suggestion
      "data": "Font Awesome Free",
```
Based on other entries, this should be the name of the icons in use.